bot/src/main/kotlin/org/epilink/bot/rulebook/RulebookLoader.kt

```diff
@@ -97,6 +97,9 @@ internal suspend fun compileRules(source: SourceCode): CompiledScript = withCont
     val compileConfig = createJvmCompilationConfigurationFromTemplate<RulebookScript> {
         jvm {
             dependenciesFromCurrentContext(wholeClasspath = true)
+
+            compilerOptions.append("-jvm-target")
+            compilerOptions.append("11")
```

Comment:
Shouldn't this be `-jvm-target 11`? Also, would it be possible to have a test on this?

Reply:
No this would act like "-jvm-target 11" as a single parameter which is not
Wdym by "a test" ? This was only added to match the jvm target of the project itself, so that inline functions could work (Kotlin can't inline JVM 11 bytecode in JVM 8 bytecode)

Reply:
Test added